### PR TITLE
cleanup(build): some bazel related cleanup

### PR DESCRIPTION
**Pull request by mmalerba**
- Removed unused dep in e2e-app
- Updates rules_sass

### CODE REVIEW

File: .bazelignore [hunk 1]
```diff
@@ -1 +1,3 @@
 node_modules
+tools/dashboard/node_modules
+tools/dashboard/functions/node_modules
```
Comment (devversion) [inline .bazelignore]: Dashboard? 🤔

Comment (mmalerba) [reply .bazelignore]: Yeah I don't know where this came from, there's nothing under `tools/dashboard` other than `node_modules` directories.

I was seeing some issues when running `bazel build ...` that seemed to be fixed by the highlight.js stuff I added

**Comment by mmalerba** [comment]
#15649 resolved the highlight.js issue, reverted the change in this PR

**Comment by mmalerba** [comment]
ahh that explains it, will fix

**Pull request merged**